rocAL/include/readers/image/image_reader.h

```diff
@@ -80,6 +80,10 @@ struct ReaderConfig {
     void set_frame_step(unsigned step) { _sequence_frame_step = step; }
     void set_frame_stride(unsigned stride) { _sequence_frame_stride = stride; }
     void set_external_filemode(ExternalSourceFileMode mode) { _file_mode = mode; }
+    void set_last_batch_policy(RocalBatchPolicy last_batch_policy, bool last_batch_padded) {
```

Comment:
please combine padded cases into the enum RocalBatchPolicy so only one parameter is passed.

Reply:
RocalLastBatchPolicy is defined in rocal_api_types.h and used in all rocal API functions. RocalBatchPolicy is defined in commons.h and used internally for all loaders and readers